services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -182,3 +182,8 @@ enum class TyomaaTyyppi {
     YLEISOTILAISUUS,
     VAIHTOLAVA,
 }
+
+enum class HankeReminder {
```

Comment:
I wonder if it would be better to name these enums without number of days in the name? So that if customer wants to change the 1st reminder to be sent when completion is in 21 days instead of 14, we would not need to change database as well as code. Could these be just ordinals as REMINDER_1 (was COMPLETION_14) and REMINDER_2 (was COMPLETION_5)? Of course, if there would be a new reminder between 1 and 2 then we would need to change the database as well. So, never mind, I think this is okay as it is.

Reply:
Yeah, I don't think we can come up with names that would make sense after being redefined. I think if we would need to change the reminder from 14 to 21 days, the correct way would be to add a new reminder to the enum with some logic that we don't send the 21-day reminder if the 14-day reminder has been sent.

Btw, the reminders are named `COMPLETION_X` because there will also be a reminder when the hanke is close to being deleted.